src/zenml/zen_stores/schemas/component_schemas.py

```diff
@@ -40,6 +40,25 @@
     from zenml.zen_stores.schemas import ScheduleSchema
 
 
+from sqlalchemy.types import String, TypeDecorator
+
+
+class HexByteString(TypeDecorator):
```

Comment:
Can you explain why this was necessary? We had byte columns in the DB before right?

Reply:
Yes, I was hoping to experiment with storing hex strings instead but yes, doesn't achieve anything significant towards the goal of easy filtering. I have removed it.